src/test/isolation2/sql/vacuum_progress_row.sql

```diff
@@ -143,8 +143,16 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
 -- Resume walsender to detect mirror down and suspend at the beginning
 -- of post-cleanup taken over by a new vacuum worker.
 2: SELECT gp_inject_fault('vacuum_worker_changed', 'suspend', dbid) FROM gp_segment_configuration WHERE content > -1 AND role = 'p';
+
+-- Dispatcher shouldn't proceed until status_version in FtsVersion in FtsProbeInfo is updated, 
```

Comment:
Space. In another file the same.
```suggestion
-- Dispatcher shouldn't proceed until status_version in FtsVersion in FtsProbeInfo is updated,
```